pyproject.toml

```diff
@@ -49,6 +49,7 @@ PyTango = { version = "^9.3.6"}
 python-ldap = "^3.4.0"
 requests = "^2.31.0"
 colorama = "^0.4.6"
+pyicat-plus = ">=0.4.0"
```

Comment:
I think we need to make pyicat-plus an optional/extra dependency as not all the sites are using it

Reply:
Indeed. How do you propose we manage this? Do I think we will do `pip install mxcube-core[<site>]` which for us would be `site="esrf"`?